src/freenet/pluginmanager/PluginManager.java

```diff
@@ -883,10 +883,13 @@ public String dumpPlugins() {
 	 * look for PluginInfo for a Plugin with given classname
 	 * @param plugname
 	 * @return the PluginInfo or null if not found
-	 * @deprecated As opposed to its JavaDoc, this function will also return plugins with a matching filename, no only class name.
-	 *             To fix this ambiguity, instead use either {@link #getPluginInfoByClassName(String)} or {@link #getPluginInfoByFileName(String)}.
+     * @deprecated As opposed to its JavaDoc, this function will also return plugins with a matching
```

Comment:
That would require me to review all callers for whether they are correct under the assumption of the new JavaDoc - which is what I was precisely trying to avoid by doing it this way :)
Was trying to speed up the release of WOT by avoiding this tedious job.
ACK to leaving it as is?